portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,83 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
-  </head>
-  <body>
+</head>
+
+<body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <h1>My SPS Portfolio</h1>
     </div>
-  </body>
-</html>
+
+    <!-- Introduction -->
+    <div id="content">
```

Comment:
What is this id for? You should only have one element with a given id on a page.  If you just want styling, this should be a `class` instead.

Reply:
Yes, my intention was to use the class so that I can apply the same styling to multiply blocks.